### DISCUSSION TITLE
这种提示标签的语法是什么，是markdown格式么

### DISCUSSION THREAD

**Discussion by chendada00** [anzhiyu-c/hexo-theme-anzhiyu#215] [General]
![image](https://github.com/anzhiyu-c/hexo-theme-anzhiyu/assets/53457984/56617907-262b-419a-ac08-9b2f6d40a2ef)

**Comment by w1ndys** [comment]
https://blog.anheyu.com/posts/d50a.html#%E5%88%86%E6%A0%8F-tab

**Reply by chendada00** [reply]
非常感谢

**Reply by w1ndys** [reply]
不对，做作者写了，在同一个页面里
https://blog.anheyu.com/posts/d50a.html#%E5%88%86%E6%A0%8F-tab

**Reply by chendada00** [reply]
我也看到了😎
